runtime/bifrost-polkadot/src/hyberbridge.rs

```diff
@@ -0,0 +1,98 @@
+// This file is part of Bifrost.
+
+// Copyright (C) Liebi Technologies PTE. LTD.
+// SPDX-License-Identifier: GPL-3.0-or-later WITH Classpath-exception-2.0
+
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see <https://www.gnu.org/licenses/>.
+
+use crate::governance::TechAdminOrCouncil;
+use crate::{Balances, Ismp, IsmpParachain, Runtime, RuntimeEvent, Timestamp};
+use bifrost_primitives::Balance;
+use frame_support::parameter_types;
+use hyperbridge_client_machine::HyperbridgeClientMachine;
+use ismp::{host::StateMachine, module::IsmpModule, router::IsmpRouter};
+use pallet_ismp::NoOpMmrTree;
+use sp_std::boxed::Box;
+use sp_std::vec::Vec;
+
+impl pallet_hyperbridge::Config for Runtime {
+	type RuntimeEvent = RuntimeEvent;
+	// pallet-ismp implements the IsmpHost
+	type IsmpHost = Ismp;
+}
+
+impl pallet_ismp_host_executive::Config for Runtime {
+	type RuntimeEvent = RuntimeEvent;
+	type IsmpHost = Ismp;
+}
+
+parameter_types! {
+	// The hyperbridge parachain on Polkadot
+	pub const Coprocessor: Option<StateMachine> = Some(StateMachine::Kusama(4009));
+	 // The host state machine of this pallet, your state machine id goes here
+	pub const HostStateMachine: StateMachine = StateMachine::Kusama(2030); // polkadot
+}
```

Comment:
Maybe you can use feature flags to select the right state machine ids for testnet/mainnet

Reply:
A separate Runtime will be made later